horovod/mxnet/adapter.cc

```diff
@@ -53,13 +53,13 @@ MXPersistentBuffer::AccessData(std::shared_ptr<OpContext> context) const {
   return buffer_;
 }
 
-template <class T> MXTensor<T>::MXTensor(T* tensor) : tensor_(tensor) {}
+MXTensor::MXTensor(NDArray* tensor) : tensor_(tensor) {}
```

Comment:
What happens if it's MXNet numpy array ?

Reply:
Is there a Numpy array class in MXNet? I think it still uses NDArray to store tensors.